scripts/core/superdesk-api.d.ts

```diff
@@ -46,6 +46,7 @@ declare module 'superdesk-api' {
         priority?: IDisplayPriority;
         label: string;
         icon: string;
+        group?: string;
```

Comment:
Can you rename this to `groupLabel` and output it as a title of the icon for accessibility? It would also be useful to be able to set a group icon. We could have the interface be `group: {label: string: icon: string};`